ci/install.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Please make this script executable and change this shebang to be `#!/bin/bash -ex`